tests/main/microk8s-smoke/task.yaml

```diff
@@ -66,3 +66,6 @@ execute: |
 
     echo "Test if nginx can be connected"
     retry -n 15 sh -c "curl --max-time 3 'http://localhost:31313/' | MATCH 'Welcome to nginx'"
+
+    echo "Running a microk8s causes no DENIED messages"
+    dmesg | not grep DENIED
```

Comment:
you could use NOMATCH instead

Reply:
you could match here a denial from a different test, not sure if we could filter the messages for the specific test

Reply:
Yeah, was thinking the same, maybe this needs a invariant check or maybe we need to filter out any old denials in the `snap run` test

Reply:
@mvo5 I think we can run "dmesg --read-clear" while the test is being prepared, so we know the old dmesg data is not displayed in the current test, does it make sense?